packages/network-jsonapi/tsconfig.build.json

```diff
@@ -14,11 +14,11 @@
     "noImplicitAny": false,
     "noImplicitReturns": true,
     "noUnusedParameters": true,
-    "outDir": "./dist",
+    "outDir": ".",
     "strict": true,
     "strictFunctionTypes": false,
-    "target": "es5",
-    "resolveJsonModule": true
+    "target": "es6",
```

Comment:
TODO: most other configs use es5 target. Should we maybe bump all of them to es6?